### DISCUSSION TITLE
Typo error

### DISCUSSION THREAD

**Discussion by vivek-m-gowda** [Sunbird-inQuiry/Community#62] [Documentation Issues]
In the 5.4.0 release there is a typo mistake in README.md file https://github.com/project-sunbird/knowledge-platform/blob/release-5.4.0/README.md

![Screenshot from 2023-02-14 10-36-45](https://user-images.githubusercontent.com/103198978/218644488-c3fb2117-d7fb-42d9-b9d9-5378db82531b.png)

It has to be **sh ./local-setup.sh**

**Comment by krgauraw** [comment]
@vivek-m-gowda This document is not related to inQuiry. So please raise the request to Knowlg Community.
cc: @PoojaJManjooran

**Reply by surendrasinghs** [reply]
Thanks, Vivek, for highlighting the typo. I have raised a PR for the same https://github.com/project-sunbird/knowledge-platform/pull/927. 

I encourage you to contribute as well.
